libcloud/test/compute/test_onapp.py

```diff
@@ -0,0 +1,111 @@
+import unittest
+import sys
+import json
+
+from mock import call, MagicMock
+
+from libcloud.test.file_fixtures import ComputeFileFixtures
+from libcloud.compute.drivers.onapp import OnAppNodeDriver
+from libcloud.test import LibcloudTestCase
+from libcloud.test.secrets import ONAPP_PARAMS
+from libcloud.compute.base import Node
+
+
+class OnAppNodeTestCase(LibcloudTestCase):
+    def setUp(self):
+        def _request(*args, **kwargs):
```

Comment:
Also, just a quick heads up for the future - we have a framework in place for mocking HTTP responses (pretty much every driver tests use it).

So in the future, please use follow the approach other tests use.